stetho/src/main/java/com/facebook/stetho/inspector/database/DatabasePeerManager.java

```diff
@@ -30,18 +29,43 @@
   };
 
   private final Context mContext;
+  private final DatabaseFilesProvider mDatabaseFilesProvider;
 
-  public DatabasePeerManager(Context context) {
+  /**
+   * Constructs the object with a {@link DatabaseFilesProvider} that supplies the database files
+   * from {@link Context#databaseList()}.
+   *
+   * @param context the context
+   */
+  public DatabasePeerManager(final Context context) {
+    this(context, new DatabaseFilesProvider() {
```

Comment:
I'm not typically a fan of hiding the default logic that way, but my sense of preserving the API won the battle.

I'll create a default implementation and deprecate existing constructor.